sklearn/tests/test_pipeline.py

```diff
@@ -214,6 +219,16 @@ def test_pipeline_fit_params():
     assert_true(pipe.named_steps['transf'].b is None)
 
 
+def test_pipeline_score_params():
+    # Pipeline should pass score parameters
+    X = np.array([[1, 2]])
+    pipe = Pipeline([('transf', Transf()), ('clf', FitParamT())])
+    pipe.fit(X, y=None)
+    assert_equal(pipe.score(X), 3)
+    assert_equal(pipe.score(X, y=None), 3)
+    assert_equal(pipe.score(X, sample_weight=np.array([2, 3])), 8)
```

Comment:
That makes sense but are there snippets of the error message that would be common across the versions? Something like `unexpected.*sample_weight` that would surely be present in all the versions?

(`assert_raises_regexp`)?